### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by thefat32** [RobinCK/typeorm-fixtures#199]
| Software         | Version(s) |
| ---------------- | ---------- |
| typeorm-fixtures | 3.0.0
| Node             | v16.14.2
| npm/Yarn         | yarn 1.22.15
| Operating System | Windows 11

The lib is unable to find fixture files inside a folder in Windows beacuse `glob.sync` seems not to work properly in glob `8`

Workaround using yarn `resolutions`:

```
  "resolutions": {
    "typeorm-fixtures-cli/**/glob": "7.2.3"
  }
```

**Comment by efiago** [comment]
I had the same problem. 
I temporarily solved with npm like this:

Add to package.json:

`"overrides": {
    "typeorm-fixtures-cli": {
      "glob": "7.2.3"
    }
  }`

**Comment by RobinCK** [comment]
fixed in 3.0.2
